In [1]:
name = "Alex"
hourly_wage = 20.00
hours = 32

weekly_pay = hourly_wage * hours

print(f"{name} earns {weekly_pay} per week")

Alex earns 640.0 per week


In [2]:
temps = [61, 65, 72, 58, 70, 74, 68]

warm_days = 0

for temp in temps:
    if temp > 65:
        warm_days = warm_days + 1

print(warm_days)

4


In [4]:
def to_celsius(f):
    return (f - 32) * 5/9

print(round(to_celsius(temps[0]), 1))

16.1


In [5]:
movie = {
    "title": "The Matrix",
    "year": 1999,
    "rating": 8.7
}

for key, value in movie.items():
    print(key, ":", value)

title : The Matrix
year : 1999
rating : 8.7


In [10]:
import pandas as pd
df=pd.read_csv("chickwts.csv")
df.head()

,Unnamed: 0,weight,feed
0,1,179,horsebean
1,2,160,horsebean
2,3,136,horsebean
3,4,227,horsebean
4,5,217,horsebean


In [11]:
df.shape

(71, 3)

In [12]:
df.describe()

,Unnamed: 0,weight
count,71.000000,71.000000
mean,36.000000,261.309859
std,20.639767,78.073700
min,1.000000,108.000000
25%,18.500000,204.500000
50%,36.000000,258.000000
75%,53.500000,323.500000
max,71.000000,423.000000


## Step 6: Interpretation of `describe()`
In the total of 71 chickens in the database, the average chicken weighs about 261. From the heaviest, weighing 423, to the lightest, weighing 108, the 315-pound difference suggests the feed nutrition varies more. 